src/pages/Home/Content/Content.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { ContentStyle } from "./Content.style";
+import { FoodItem, Category } from "DummyData/DataTypes";
+import { mcSpicy } from "../../../DummyData/mcSpicy";
+import { ovaltineMcFlurry } from "../../../DummyData/ovaltineMcFlurry";
+import { categories } from "../../../DummyData/categories";
+import SectionHeader from "./Section/SectionHeader/SectionHeader";
+import ItemList from "./Section/ItemList/ItemList";
+
+const AllFoodItems: FoodItem[] = [
+  mcSpicy,
+  mcSpicy,
+  mcSpicy,
+  mcSpicy,
+  mcSpicy,
+  mcSpicy,
+  ovaltineMcFlurry,
+  ovaltineMcFlurry,
+  ovaltineMcFlurry,
+];
+
+//trying to render 6 cards of identical McSpicy foodItems. Should be under the "recommended" SectionHeader.
+//McSpicy would appear both under "Recommended" and "Main Course". FoodItem can have more than one category.
+export default function Content() {
+  return (
+    <ContentStyle>
+      {categories.map((sectionHeader) => {
+        const foodItemToRender: FoodItem[] = [];
```

Comment:
You should not write the logic into the return value of the component. Do it above and then map through the prepared data to generate the components